test/index.html

```diff
@@ -1,9 +1,36 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>DNI testing</title>
-</head>
-<body>
-	<h1>SpanishIDValidator testing</h1>
-</body>
-</html>
\ No newline at end of file
+
+  <head>
+    <!-- jasmine -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jasmine/1.3.1/jasmine.js"></script>
+    <!-- jasmine's html reporting code and css -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jasmine/1.3.1/jasmine-html.js"></script>
+    <link href="//cdnjs.cloudflare.com/ajax/libs/jasmine/1.3.1/jasmine.css" rel="stylesheet" />
+    <!-- angular itself -->
+    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.11/angular.js"></script>
```

Comment:
Better use latest version of Angular: 1.3.14 (I checked and it's working fine with Jasmine).